packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/overview-identity/overview-identity-edit/overview-identity-edit.constants.js

```diff
@@ -0,0 +1,8 @@
+export const REGEX_VALIDATORS = {
+  SIRET: /^\d{14}$/,
+  APE: /^\d{4}[a-zA-Z]$/,
+};
+
+export default {
+  REGEX_VALIDATORS,
+};
```

Comment:
Seems to be a duplicate of https://github.com/ovh/manager/blob/feat%2FMANAGER-8528/packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/form/form.constants.js